assets/js/editor-components/block-title/index.js

```diff
@@ -21,22 +21,19 @@ const BlockTitle = ( {
 } ) => {
 	const TagName = `h${ headingLevel }`;
 	return (
-		<TagName>
+		<TagName
+			className={ classnames(
+				'wc-block-editor-components-title',
```

Comment:
Sorry if I wasn't clear here. I think we should keep `wc-block-editor-components-title` class into the `PlainText` component. That class is used to make the textarea background transparent:

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/0811719ed486c3952a1e022490b3715766379f77/assets/js/editor-components/block-title/editor.scss#L1-L6

Reply:
> Sorry if I wasn't clear here. I think we should keep wc-block-editor-components-title class into the PlainText component. That class is used to make the textarea background transparent.

My bad as well. I just adjusted this section based on your feedback. The CSS class for the `<Tagname>` will be injected via prop, while the element `<PlainText>` uses the hard-coded CSS class `.wc-block-editor-components-title`.